### PR TITLE
Alert yml  config

### PR DESCRIPTION
**Pull request by zqr10159**
## What's changed?

Alert  yml  config

## Checklist

- [x]  I hereby agree to the terms of the [HertzBeat CLA](https://gist.github.com/tomsun28/511c04e7643901cb550bb6ecc75a661b)
- [ ]  I have read the [Contributing Guide](https://hertzbeat.com/docs/others/contributing/)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](../e2e) and all cases have passed.

### CODE REVIEW

**Comment by tomsun28** [comment]
👍，hi建议不把不同环境分离出来做成独立yml，因为docker挂载的时候就需要再挂载一个，但是我们对外文档说明一直都是挂载那个 application.yml，改了之后会造成疑虑

**Comment by hertzbeat** [comment]
> Bot detected the issue body's language is not English, translate it automatically. 👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿         
----    
👍, Hi, it is not recommended to separate different environments into independent ymls, because when docker is mounted, another one needs to be mounted. However, our external documentation always states that the application.yml is mounted. Changing it will cause doubts.

File: alerter/src/main/java/org/dromara/hertzbeat/alert/AlerterProperties.java [hunk 1]
```diff
@@ -35,37 +35,37 @@ public class AlerterProperties {
...
      * Discord Notify url
      */
-    private String discordNotifyUrl = "https://discord.com/api/v9/channels/%s/messages";
+    private String discordNotifyUrl;
 
     /**
      * ServerChan Notify url
      */
-    private String serverChanNotifyUrl = "https://sctapi.ftqq.com/%s.send";
+    private String serverChanNotifyUrl;
```
Comment (tomsun28) [inline alerter/src/main/java/org/dromara/hertzbeat/alert/AlerterProperties.java:68]: 建议也在代码中写好默认值url，这样用户没有在application.yml配置也不会出错

File: manager/src/main/resources/application-dev.yml [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  telegram-bot-api-url: https://api.telegram.org/bot%s/sendMessage
+  # discord
+  discord-notify-url: https://discord.com/api/v9/channels/%s/messages
+  # server酱
+  server-chan-notify-url: https://sctapi.ftqq.com/%s.send
+
+scheduler:
+  server:
+    enabled: true
+    port: 1158
```
Comment (tomsun28) [inline manager/src/main/resources/application-dev.yml:184]: debug dev环境的profile留在自己本地就行，我们在 gitinore 已经将其排除 

image

**Comment by zqr10159** [comment]
ok

**Review by tomsun28 — APPROVED** [review]
LGTM!👍👍

**Pull request merged**